dind-aws/Dockerfile

```diff
@@ -25,6 +25,10 @@ RUN echo "Install AWS" && \
     chmod +x /usr/local/bin/docker-compose && \
     docker-compose --version && \
 
+    sed -e "s,\$DOCKER_PORT_2375_TCP,\$EKINO__DOCKER_BUILDBOX_PORT_2375_TCP," \
+        -e "s,export DOCKER_HOST='tcp://docker:2375',export DOCKER_HOST='tcp://ekino__docker-buildbox:2375'," \
+        /usr/local/bin/docker-entrypoint.sh && \
```

Comment:
To edit the file in-place, you need to call `sed` with the `-i` option as well.

Reply:
Oops good catch, forgot it, thx